src/modules/reserve-overview/ReserveConfiguration.tsx

```diff
@@ -42,8 +41,8 @@ export const ReserveConfiguration: React.FC<ReserveConfigurationProps> = ({ rese
   const renderCharts =
     !!currentNetworkConfig.ratesHistoryApiUrl && !BROKEN_ASSETS.includes(reserveId);
   const { supplyCap, borrowCap, debtCeiling } = useAssetCaps();
-  const showSupplyCapStatus = reserve.supplyCap && reserve.supplyCap !== '0';
-  const showBorrowCapStatus = reserve.borrowCap && reserve.borrowCap !== '0';
+  const showSupplyCapStatus = reserve.supplyCap && reserve.supplyCap !== '0' ? true : false;
+  const showBorrowCapStatus = reserve.borrowCap && reserve.borrowCap !== '0' ? true : false;
```

Comment:
This seems unnecessary.  What's the reasoning?  We can always use a type declaration instead of inference for more clarity.

Reply:
It was because `reserve.borrowCap && reserve.borrowCap !== '0'` evaluated to `string | boolean` 

I realized the first condition is actually unnecessary anyways, so I removed it and declared the type as boolean without that ugly ternary